### ISSUE TITLE
`explain` doesn't work on the Analyze API

### ISSUE THREAD

**Issue by ackintosh** [WorksApplications/elasticsearch-sudachi#142]
Elasticsearch 8.6.2
analysis-sudachi 3.1.0

Got an error when set `explain` to true:

```bash
curl -X GET "localhost:9200/_analyze?pretty" -H 'Content-Type: application/json' -d '{"tokenizer":"sudachi_tokenizer", "text" : "関西国際空港", "explain": true}'

{
  "error" : {
    "root_cause" : [
      {
        "type" : "illegal_argument_exception",
        "reason" : "cannot write xcontent for unknown value of type class com.worksap.nlp.sudachi.MorphemeImpl"
      }
    ],
    "type" : "illegal_argument_exception",
    "reason" : "cannot write xcontent for unknown value of type class com.worksap.nlp.sudachi.MorphemeImpl",
    "suppressed" : [
      {
        "type" : "illegal_state_exception",
        "reason" : "Failed to close the XContentBuilder",
        "caused_by" : {
          "type" : "i_o_exception",
          "reason" : "Unclosed object or array found"
        }
      }
    ]
  },
  "status" : 400
}
```

**Comment by mh-northlander** [comment]
elasticsearch-sudachi ~v3.1.* has a bug around `explain` (#102) and it is fixed in v3.2.0.

**Comment by ackintosh** [comment]
Oh, sorry for the duplication. Thanks!
